HW04/maksim_manzhai/lib/homework.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Homework
+  attr_reader :title, :mentor, :student
+  attr_accessor :answers, :acceptable
+
+  def initialize(title:, description:, student:, mentor:)
+    @title = title
+    @description = description
+    @student = student
+    @mentor = mentor
+    send_notification_to_student
+    @answers = []
+    @acceptable = false
+  end
+
+  def send_notification_to_student
```

Comment:
Why do you need this method?

Reply:
we need send notification to student about new homework

Reply:
so, you can do it in your `add_homework` method :)